README.md

```diff
@@ -19,238 +20,67 @@ This is the base Nerves System configuration for [QEMU](http://wiki.qemu.org/Mai
 | WiFi                 | No                              |
 | Bluetooth            | No                              |
 
-This is the start of a QEMU configuration. Instructions are given below
-to make networking work on Linux and OS X QEMU hosts.
+Unlike other Nerves systems, this one does not contain a minimal configuration
+since it is expected to be used exclusively for debugging on a fast computer.
+Currently, it contains the following additional packages:
 
-## Booting
+* The `qt-webkit-kiosk` fullscreen browser
 
-### Hello, Nerves!
+As such, it takes a long time to build if you need to make custom modifications
+to it. The goal is that this is uncommon. However, if this does apply to you,
+you may want to trim now the configuration while experimenting with itso that
```

Comment:
`itso` missing a space.
Maybe we should talk about how it requires more that 4gb of ram to build